cni/plugin.go

```diff
@@ -20,8 +20,11 @@ import (
 	cniTypes "github.com/containernetworking/cni/pkg/types"
 	cniTypesCurr "github.com/containernetworking/cni/pkg/types/current"
 	cniVers "github.com/containernetworking/cni/pkg/version"
+	"github.com/pkg/errors"
 )
 
+var errEmptyContent = errors.New("Read content is zero bytes")
```

Comment:
error strings should begin lowercase
```suggestion
var errEmptyContent = errors.New("read content is zero bytes")
```